odoo/http.py

```diff
@@ -1268,12 +1268,16 @@ def start_wrapped(status, headers):
             req = werkzeug.wrappers.Request(environ)
             root.setup_session(req)
             if req.session and req.session.debug and not 'wkhtmltopdf' in req.headers.get('User-Agent'):
-                new_headers = [('Cache-Control', 'no-cache')]
+                cache_control_value = 'no-cache'
+                new_headers = []
 
                 for k, v in headers:
                     if k.lower() != 'cache-control':
                         new_headers.append((k, v))
+                    else:
+                        cache_control_value += ', %s' % v
```

Comment:
Should probably check if it already contains `no-cache` just in case (and just keep the existing value)

Reply:
I think that's the wrong way around: this is `<current value> not in 'no-cache'` so let's say the current response has a `Cache-Control: private, no-cache`, `private, no-cache` is not in `no-cache`, therefore false, therefore we'll prepend `no-cache`, ending with `no-cache, private, no-cache`.

Reply:
Thanks for your feedback.
I didn't consider this case. I fixed it.
Please let me know if there is anything else.

Reply:
Seems fine now :+1:

Reply:
@pedrambiria seems good to me too, don't forget to update the commit message :)

Reply:
@xmo-odoo , @Julien00859 Hello,
I did update the commit message.
Could you please r+ it?